docs/latest/design/ratelimit.md

```diff
@@ -150,11 +142,7 @@ spec:
   hostnames:
   - www.example.com
   rules:
-  - clientSelectors:
```

Comment:
This shouldnt be deleted, instead needs to be renamed to `matches`
I had done a  `s/matches/clientSelectors/g' on the entire file 🙈